modules/sb-configuring-asb-monitoring.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/configuring-ansible-service-broker.adoc
+
+[id="sb-configuring-monitoring-asb_{context}"]
+= Configuring monitoring for the {asb-name}
+
+In order for Prometheus to monitor the {asb-name}, you must create the following resources to grant Prometheus permission to access the namespace where the {asb-name} was installed.
+
+[NOTE]
+====
```

Comment:
Instead of a NOTE, could this be listed as a Prerequisite?

Reply:
@ahardin-rh you know, my note was intending to say that the procedure is assuming you installed it into X namespace, but that they really could have installed it in a different one. but you're right that generically having it installed at all is a prereq, i will definitely add that. thanks for the suggestion!